app/code/Magento/Catalog/Block/Product/View/Attributes.php

```diff
@@ -84,13 +84,15 @@ public function getAdditionalData(array $excludeAttr = [])
 
                 if (!$product->hasData($attribute->getAttributeCode())) {
                     $value = __('N/A');
+                } elseif ($value instanceof Phrase) {
+                    $value = (string)$value;
                 } elseif ((string)$value == '') {
```

Comment:
Please remove this `elseif` as converting an empty value to `No` is pretty weird logic. If this is a workaround for some bug it should be fixed in another way as this code only hides the real problem.